pkg/tkctl/config/config.go

```diff
@@ -14,17 +14,18 @@
 package config
 
 import (
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path/filepath"
+	"sync"
+
 	"github.com/golang/glog"
 	"gopkg.in/yaml.v2"
-	"io/ioutil"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/genericclioptions/resource"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-	"os"
-	"os/user"
-	"path/filepath"
-	"sync"
```

Comment:
imports changed because the code is formatted by `goimports` tool. It automatically puts the standard packages at the front of the import list.